javascript/chronometer.js

```diff
@@ -1,37 +1,67 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      if (callback) {
+        callback();
+      }
+      this.currentTime++;
+    }, 1000);
+    return this.currentTime;
```

Comment:
good job Nico, but it was not necessary to return this.currentTime in the start function. And if you don't return it, you could have used this.currentTime instead of this.start() later, e.g. this.currentTime % 60.

Reply:
Thanks - this.start() give me a headache in part 2.